### ISSUE TITLE
Separate @Column (indexed = true) for methods creation and sqlite index creation

### ISSUE THREAD

**Issue by jozn** [gfx/Android-Orma#282]
Hi, i am switching to this library, from DBFlow.
I a'm creating some table, i want to query some columns without the need to force sqlite make index on them(have methods in java but not index on table).  sometimes i want to query all fields , and many fields contains few values (ex: 0 or 1). making index on all of fields is crazy. Generating methods for querying a database  is not the same of having index on fields, this is two different things, and i think this should be separated.

A quick fix for this and to be backward compatible maybe be something like:   
```

@Column (indexed = true)// both indexed, and gen methods
String userName;

@Column (methods = true)// just generate methods
String gender;

```
I like to see this in next release of this nice lib.

**Comment by gfx** [comment]
Thanks to the indication. It seems good to me and will implement it in a few days.

I think the interface is something like this: `@Column(helpers = Helpers.ALL)` where `.ALL`, `.NONE`, `.FILTERS`, `.ORDERS`, and so on.

**Comment by jozn** [comment]
Thank you for your respond and effort.
I have also used SquiDB, but those library was so abstract, and complex and hard to use and did't feel what your final query would look like. I have read your library code and generated classes, it easy to warp my mind around it, it has nice design.
Thanks.

**Comment by gfx** [comment]
This feature is released as v2.6.0. Can you give it a try?

**Comment by jozn** [comment]
Yeh, i have tested this, and it works as expected, thanks.
